devicetypes/stelpro/stelpro-maestro-thermostat.src/stelpro-maestro-thermostat.groovy

```diff
@@ -149,29 +149,29 @@ def setupHealthCheck() {
 	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 }
 
-def installed() {
-	setupHealthCheck()
-
+def configureSupportedRanges() {
 	sendEvent(name: "supportedThermostatModes", value: supportedThermostatModes, displayed: false)
 	sendEvent(name: "thermostatSetpointRange", value: thermostatSetpointRange, displayed: false)
 	sendEvent(name: "heatingSetpointRange", value: heatingSetpointRange, displayed: false)
 }
 
+def installed() {
+	setupHealthCheck()
+
+	configureSupportedRanges()
+}
+
 def updated() {
 	def requests = []
 
 	setupHealthCheck()
```

Comment:
setupHealthCheck() and configureSupportedRanges() are in installed(). Maybe just call it.